R/vars-select.R

```diff
@@ -229,20 +243,36 @@ inds_combine <- function(vars, inds) {
 }
 
 inds_unique <- function(x) {
-  # Remove duplicates
-  out <- vctrs::vec_unique(x)
+  # Remove duplicates but keep last name of duplicates
+  split <- vctrs::vec_split(x, x)
+  out <- split$key
 
   # Keep last name of duplicates
   if (length(out) < length(x)) {
-    reversed <- rev(x)
-    rev_unique_locs <- vctrs::vec_unique_loc(reversed)
-    unique_nms <- rev(names2(reversed)[rev_unique_locs])
-    names(out) <- unique_nms
+    names(out) <- map_chr(split$val, ind_last_name)
   }
 
   out
 }
 
+ind_last_name <- function(x) {
+  names <- names(x)
+  last <- last(names)
+
+  if (!all(vctrs::vec_duplicate_detect(names) | names == "")) {
+    dups <- encodeString(names, quote = "\"")
```

Comment:
I think this should be `` ` ``? (Since it's a variable name not a string)

Reply:
oh of course